cmd/timestamp-server/app/root.go

```diff
@@ -70,7 +70,8 @@ func init() {
 	rootCmd.PersistentFlags().String("file-signer-key-path", "", "Path to file containing PEM-encoded private key. Supported formats include PKCS#1, PKCS#8, and RFC5915 for EC")
 	rootCmd.PersistentFlags().String("file-signer-passwd", "", "Password to decrypt private key")
 	// NTP time introspection
-	rootCmd.PersistentFlags().String("ntp-monitoring", "", "Path to a file configuring ntp monitoring")
+	rootCmd.PersistentFlags().String("ntp-monitoring", "ntpsync.yaml", "Path to a file configuring ntp monitoring. Defaults to ntpsync.yaml in the root dir")
```

Comment:
Q: why do we default to a specific filename ? How do we get that configuration file ?

What about defaulting to some hardcoded settings and change those if the user provides a different path to a config file ?

Reply:
cc @haydentherapper

Reply:
I wouldn't expect users to configure their own list typically. It's a curated list from many different sources. Ideally we'd just use the file checked into the repo.

IIRC though, golang doesn't embed files from source in the binary, so I don't think this will work. I think you'll need to use https://pkg.go.dev/embed also.

Reply:
When I run the server locally, it seems to read the file correctly. However, I can use https://pkg.go.dev/embed

Reply:
It might be the difference between go run and building the binary? But I'm not sure